src/Directory.Packages.props

```diff
@@ -69,7 +69,7 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageVersion Include="TypeNameFormatter.Sources" Version="1.0.1" />
+    <PackageVersion Include="TypeNameFormatter.Sources" Version="1.1.0" />
```

Comment:
I put 1.1.1 with the `[SuppressMessage]` thingy on NuGet just now. Hopefully that'll make the workaround unnecessary.